README.md

```diff
@@ -8,45 +8,274 @@ native Bitcoin capabilities of creating multisig transactions.
+* [Example code](examples/btc_anchoring.rs)
+* [Deployment guide](#deployment)
 * [Contribution guide][exonum:contribution]
 
-## Installation guide
+## Prerequisites
 
-Just follow the installation guide of the [`exonum`][exonum:install] to
-install dependencies.
+### Installation
+
+Just follow [Exonum][exonum:install] installation guide to install dependencies.
+
+### Bitcoin Node Deployment
+
+First of all install `bitcoind` via your package manager and ensure that you
+use the latest stable version. You may visit official Bitcoin [site][bitcoin:install]
+for more information about installation.
+
+Then create `bitcoind` configuration file according to this [tutorial][bitcoin_wiki:configuration].
+
+For correct work of the service, the `bitcoind` configuration file
+should contain the following settings:
+
+```ini
+# Run the node in the test network instead of the real Bitcoin network.
+# If you want to use the main network comment the line bellow:
+testnet=1
+# server=1 tells `Bitcoin-Qt` and `bitcoind` to accept JSON-RPC commands.
+server=1
+# Maintain a full transaction index, used by the `getrawtransaction` RPC call.
+# An arbitrary `bitcoind` daemon is not required to respond to a request for
+# information about an arbitrary transaction, thus you should uncomment the
+# line bellow if you want to use the daemon in an existing Exonum network.
+# txindex=1
+
+# Bind to the given address to listen to JSON-RPC connections.
+# Use [host]:port notation for IPv6.
+# This option can be specified multiple times (default: bind to all interfaces)
+#rpcbind=<addr>
+# You must specify `rpcuser` and `rpcpassword` to secure the JSON-RPC API
+#rpcuser=<username>
+#rpcpassword=YourSuperGreatPasswordNumber_DO_NOT_USE_THIS_OR_YOU_WILL_GET_ROBBED_385593
+```
+
+These RPC settings will be used by the service.
+
+After creating configuration file, launch `bitcoind` daemon via command:
+
+```shell
+bitcoind --daemon
+```
+
+***Note!** Downloading and indexing of the Bitcoin blockchain may take a lot of time,
+especially for the mainnet.*
 
 ## Usage
 
-The anchoring service depends on bitcoind. For the correct work, you need to
-launch bitcoind with specific configuration, see [deployment guide](DEPLOY.md)
-for details.
+Include `exonum-btc-anchoring` as a dependency into your `Cargo.toml`:
+
+```toml
+[dependencies]
+exonum-btc-anchoring = "0.10"
+```
+
+Add the BTC anchoring service to the blockchain in the main project file:
+
+```rust
+extern crate exonum;
+extern crate exonum_btc_anchoring as anchoring;
+
+use exonum::helpers;
+use exonum::helpers::fabric::NodeBuilder;
+
+fn main() {
+    exonum::crypto::init();
+    helpers::init_logger().unwrap();
+    let node = NodeBuilder::new()
+        .with_service(Box::new(anchoring::ServiceFactory));
+    node.run();
+}
+
+```
+
+## Configuration Parameters
+
+### For the `generate-template` subcommand
+
+* `btc-anchoring-network` - Bitcoin network type used for downloading Bitcoin blocks headers.
```

Comment:
@alekseysidorov please add these changes to the changelog. I mean that now parameters start with *btc-* prefix